src/modules/twap/containers/TwapFormWidget/index.tsx

```diff
@@ -47,6 +56,59 @@ export function TwapFormWidget() {
     isCustomDeadline,
   }
 
+  const LABELS_TOOLTIPS: LabelTooltipItems = {
```

Comment:
Could you please move this const to a separated file? Like `tooltips.tsx`

Reply:
Yes, for sure. I think the file is too long now. Should I create this in the same folder? Or you think it's good to make it part of global constants?